android-components/components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/AddonFilePickerTest.kt

```diff
@@ -85,4 +86,14 @@ class AddonFilePickerTest {
 
         verify(addonManager).installAddon(any<String>(), any(), any())
     }
+
+    @Test
+    fun `WHEN creating an intent from a AddonOpenDocument THEN it contains mime types for zips and xpi files`() {
+        val contractResult = AddonOpenDocument()
+
+        val intent = contractResult.createIntent(testContext, arrayOf())
+
+        val mimeTypes = intent.extras!!.getStringArray(Intent.EXTRA_MIME_TYPES)
+        assertTrue(mimeTypes!!.all { it!! == "application/x-xpinstall" || it == "application/zip" })
```

Comment:
This assertion will still pass if we remove one of the two mime types in `AddonFilePicker`, that isn't great. Let's verify the whole list of mime types instead:

```suggestion
        assertArrayEquals(mimeTypes, arrayOf("application/x-xpinstall", "application/zip"))
```

Reply:
Thanks for the suggestion, I updated the PR with it :)